Cargo.lock

```diff
@@ -403,9 +403,9 @@ dependencies = [
 [[package]]
 name = "kvproto"
 version = "0.0.1"
-source = "git+https://github.com/pingcap/kvproto.git#e4374c17d36545deaa9f4bbccb0d7599b1f05cae"
+source = "git+https://github.com/huachaohuang/kvproto.git?branch=import-sst#77985b6bbfc8f6c04f6f17c403462f09f875b2eb"
```

Comment:
can we use master here?

Reply:
Yes, but we need to merge this first https://github.com/pingcap/kvproto/pull/219.